omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -58,6 +59,31 @@ def solr_root
     host
   end
 
+  def elastic_search_major_version
```

Comment:
Fixed to raise exceptions when faced with invalid data.  Also does retries on flaky connections.